lib/devcycle-ruby-server-sdk/models/user_data.rb

```diff
@@ -414,5 +411,9 @@ def _to_hash(value)
         value
       end
     end
+
+    def to_json
+      Oj.dump(to_hash, mode: :json)
```

Comment:
Do we want to apply this to all calls to dump?

Reply:
yeah that's a good call. Only classes that have a hash as an attribute will need it tho. I'll make the changes